### ISSUE TITLE
SoundCloud clips with no artwork causes error

### ISSUE THREAD

**Issue by jacousteau** [oscarotero/Embed#52]
Hi

I found that embedding Soundcloud clips without artwork causes an error -

Example clip: https://soundcloud.com/ididthat-1/lowe-ct-cape-times-world-cup

Error
vendor/embed/embed/src/Adapters/Soundcloud.php(89): Embed\Utils::unshiftValue(Array, Array)

'Illegal offset type' in vendor/embed/embed/src/Utils.php:135

Note, when I comment out the following section in Adapters/Soundcloud.php
line: 85 - 90
```
/*
        if (!$this->api->get('artwork_url') && ($img = $this->api->get('user', 'avatar_url'))) {
            Utils::unshiftValue($images, [
                'value' => str_replace('-large.jpg', '-t500x500.jpg', $img),
                'providers' => ['api'],
            ]);
        }
*/
```

The clip renders perfect, with the Avatar as clip's artwork.

**Comment by oscarotero** [comment]
Thank you. Fixed and release 2.1.2

**Comment by jacousteau** [comment]
Awesome. Great library! thank you.
